react/src/views/Box/BoxView.tsx

```diff
@@ -232,10 +234,8 @@ function BTBox() {
     assignBoxToDistributionEventMutationStatus.error ||
     unassignBoxFromDistributionEventMutationStatus.error
   ) {
-    notificationVar({
-      title: "Error",
-      type: "error",
-      message: "Error: Could not update the box",
+    triggerError({
+      message: "Could not update the box.",
```

Comment:
Not directly related to the PR, however for `error` it would make sense to have a message like `Could not fetch box data`. For mutation errors the message above makes sense.

Reply:
changed this